docs/orbit_registration.md

```diff
@@ -0,0 +1,142 @@
+# Registering a Zenko Instance on Orbit Management UI
+
+This section of documentation will help you register your Zenko instance on
+Orbit, for either [minikube](../charts/minikube.md)-based or
+[MetalK8s](../charts/README.md)-based or
+[Docker Swarm](./swarm_production_link)-based deployments.
+
+## Minikube and MetalK8s (and other Kubernetes-based deployments)
+
+### Step 1: go to the Kubernetes dashboard
+
+##### For minikube:
+```
+$ minikube dashboard
+```
+A new tab should open in your browser, taking you to the Kubernetes dashboard.
+
+##### For MetalK8s:
+```
+$ kubectl proxy
+```
+While the tunnel is up and running, access the dashboard at
+[http://localhost:8001/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy/](http://localhost:8001/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy/).
+
+### Step 2: finding the Zenko instance ID
+
+#### Option A: from the CloudServer logs
+
+From the dashboard, click on the cloudserver-front service, and then on one of
+the pods running for that service. On the top right menu, click "Logs", and then
+do a search of "instance ID" - you should find a line in the logs giving you a
+long string of random characters: this is your instance id, copy it. If you
+cannot find such a line in the logs, try option B.
+
+#### Option B: from the Metadata MongoDB cluster
+
+From the dashboard, click on the mongodb-replicaset service, and then on the
+pod that is mongodb-replicaset-**0**. This is important as the other pods are
+slaves to that master pod 0, so you will not be able to query the MongoDB
```

Comment:
Are you sure the master is always the pod 0 ?

Reply:
This is empirical, of course, but it's always pod0 that comes up first at start time(as should the mongo master), and on my fairly numerous deployments, it's always been the master. Now I guess if you crash it, the master might change? But at start time, it should be pod 0, yes.

Reply:
This is true, pod0 will always come up first in a fresh deployment and generally be the master. However in many situations (long running stacks, VM host is rebooted, stack updated, etc) it is very likely that pod0 will no longer be the replicaset master as any of the others are able to take this role. It may be good to have a note that this is typically correct but not always the case.

Reply:
Added a "TIP" kind of note, let me know if it works for you :)